shard.yml

```
@@ -1,7 +1,8 @@
 name: zeromq
-version: 0.1.0
+version: 0.1.3
```

Let's bump (at least) a major version since this is backward-incompatible change

```suggestion
version: 0.2.0
```